front_end/panels/ai_chat/ui/settings/advanced/VectorDBSettings.ts

```
@@ -0,0 +1,290 @@
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import { i18nString } from '../i18n-strings.js';
+import {
+  VECTOR_DB_ENABLED_KEY,
+  MILVUS_ENDPOINT_KEY,
+  MILVUS_USERNAME_KEY,
+  MILVUS_PASSWORD_KEY,
+  MILVUS_COLLECTION_KEY,
+  MILVUS_OPENAI_KEY
+} from '../constants.js';
```

ESLint

[error] `../constants.js` import should occur before import of `../i18n-strings.js`

(import/order)